<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapsWindow</class>
 <widget class="QMainWindow" name="MapsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="myGraphicsView" name="graphicsView">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="qthEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>41</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="msgEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>41</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="posEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>41</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionLoad_Map"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <addaction name="menuMap"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Map"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="icon">
    <iconset>
     <normaloff>:/main/toolbar/terminal.bmp</normaloff>:/main/toolbar/terminal.bmp</iconset>
   </property>
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionStationList">
   <property name="icon">
    <iconset>
     <normaloff>:/main/toolbar/list.bmp</normaloff>:/main/toolbar/list.bmp</iconset>
   </property>
   <property name="text">
    <string>Station list</string>
   </property>
  </action>
  <action name="actionMessages">
   <property name="icon">
    <iconset>
     <normaloff>:/main/toolbar/mesg.bmp</normaloff>:/main/toolbar/mesg.bmp</iconset>
   </property>
   <property name="text">
    <string>Messages</string>
   </property>
  </action>
  <action name="actionLoadMap">
   <property name="icon">
    <iconset>
     <normaloff>:/main/toolbar/map.bmp</normaloff>:/main/toolbar/map.bmp</iconset>
   </property>
   <property name="text">
    <string>Load map</string>
   </property>
  </action>
  <action name="actionZoomIN">
   <property name="icon">
    <iconset>
     <normaloff>:/main/toolbar/z_in.bmp</normaloff>:/main/toolbar/z_in.bmp</iconset>
   </property>
   <property name="text">
    <string>Zoom IN</string>
   </property>
  </action>
  <action name="actionZoomOUT">
   <property name="icon">
    <iconset>
     <normaloff>:/main/toolbar/z_out.bmp</normaloff>:/main/toolbar/z_out.bmp</iconset>
   </property>
   <property name="text">
    <string>Zoom OUT</string>
   </property>
  </action>
  <action name="actionLoadMap_2">
   <property name="text">
    <string>Load Map</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>---</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../gui.qrc">
     <normaloff>:/images/z_in.png</normaloff>:/images/z_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../gui.qrc">
     <normaloff>:/images/z_out.png</normaloff>:/images/z_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionLoad_Map">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../gui.qrc">
     <normaloff>:/images/mopen.png</normaloff>:/images/mopen.png</iconset>
   </property>
   <property name="text">
    <string>Load Map</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>include/mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gui.qrc"/>
 </resources>
 <connections/>
</ui>
